### DISCUSSION TITLE
tcp+自动https+Proxy Protocol+nginx网页显示EOL

### DISCUSSION THREAD

**Discussion by w311ang** [natfrp/wiki#63] [使用问题] [answered]
用的是#59 绍兴多线2节点，只要关闭Proxy Protocol并在nginx中删去相关配置就可以恢复正常，在之前没有强制https之前使用Proxy Protocol都是正常的，有没有好心人能够复现一下

**Comment by fakeboboliu** [comment, marked as answer]
自动 HTTPS 为较为基础的需求设计，暂时不适用 Proxy Protocol，我们可能会在后续的版本中支持此用例。

如果需要这些复杂用法，请在 Nginx 配置 HTTPS 作为替代。

**Reply by fakeboboliu** [reply]
更新：
我们发现待办列表中为 自动 HTTPS 提供 XFF 的事项被错误地忽略了。
在添加 XFF 的同时，Proxy Protocol 的支持已经被顺便加入了。

这些更改将于下次客户端软件更新时提供。
